lib/siwapp/settings.ex

```diff
@@ -53,10 +54,22 @@ defmodule Siwapp.Settings do
       {:error, %Ecto.Changeset{}}
         # because value field is required
 
+      iex> create_series(%{default: true})
+      {:error, "You cannot directly assign..."}
+
   """
-  @spec create_series(%{optional(atom()) => any()}) ::
-          {:ok, %Series{}} | {:error, %Ecto.Changeset{}}
-  def create_series(attrs \\ %{}) do
+  @spec create_series(%{optional(any()) => any()}) ::
+          {:ok, %Series{}} | {:error, any()}
+  def create_series(attrs \\ %{})
+
+  def create_series(attrs) when is_map_key(attrs, :default) do
```

Comment:
instead of the guard I think it would be clearer a pattern matching

Reply:
It's true. I changed it in a new commit